<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="AppContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="AppPage">
     <widget class="QStackedWidget" name="Page">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>731</width>
        <height>381</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="home_page">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>661</width>
          <height>41</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>搜尋開發者或是資料包</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>711</width>
          <height>111</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="MainPageDatapacks">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>707</width>
           <height>107</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>111</width>
            <height>80</height>
           </rect>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>58</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>0</y>
            <width>111</width>
            <height>80</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>111</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>0</y>
            <width>111</width>
            <height>80</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>111</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>0</y>
            <width>111</width>
            <height>80</height>
           </rect>
          </property>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>0</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_25">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>111</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>340</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>上架資料包</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="set_user_data_page">
       <widget class="QLabel" name="background_setuserpage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>731</width>
          <height>381</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Images.qrc">:/image/Deafault_Background.png</pixmap>
        </property>
       </widget>
       <widget class="QWidget" name="userselfcolorboard" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>231</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(37, 139, 234,130) ; border-radius:25px;</string>
        </property>
        <widget class="QLabel" name="user_icon_setting">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Images.qrc">:/image/user_icon.png</pixmap>
         </property>
        </widget>
        <widget class="QTextEdit" name="setuserpage_description">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>50</y>
           <width>131</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:5;background-color:rgba(0,0,0,0);</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>因為這裡不支援我換行，我直接寫在程式</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="setpage_username">
         <property name="geometry">
          <rect>
           <x>72</x>
           <y>31</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:5;background-color:rgba(0,0,0,0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>點擊編輯使用者名稱</string>
         </property>
        </widget>
       </widget>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>711</width>
          <height>81</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1000</width>
           <height>56</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>711</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(48,54,58)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>201</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>這位開發者開發的資料包</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_31">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>251</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>設定使用者資料板的顏色</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="userdatasetR">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>37</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="userdatasetG">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>139</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="userdatasetB">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>50</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>234</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="userdatasetA">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>50</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>130</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_32">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>R(紅)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>30</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>G(綠)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_34">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>41</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>B(藍)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_35">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>A(透明度)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>340</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>更改背景圖片</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>340</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>完成設定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>340</y>
          <width>91</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>更變頭像</string>
        </property>
       </widget>
       <zorder>background_setuserpage</zorder>
       <zorder>userselfcolorboard</zorder>
       <zorder>label_27</zorder>
       <zorder>scrollArea_2</zorder>
       <zorder>label_30</zorder>
       <zorder>label_31</zorder>
       <zorder>userdatasetR</zorder>
       <zorder>userdatasetG</zorder>
       <zorder>userdatasetB</zorder>
       <zorder>userdatasetA</zorder>
       <zorder>label_32</zorder>
       <zorder>label_33</zorder>
       <zorder>label_34</zorder>
       <zorder>label_35</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>pushButton_5</zorder>
       <zorder>pushButton_6</zorder>
      </widget>
     </widget>
     <widget class="QWidget" name="top_container" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QLabel" name="usericon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>訪客</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <zorder>pushButton_2</zorder>
      <zorder>usericon</zorder>
      <zorder>label_2</zorder>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>791</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/image/AD.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>程式狀態</string>
      </property>
     </widget>
     <widget class="QLabel" name="status">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>460</y>
        <width>661</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Login_RegisterPage">
     <widget class="QLabel" name="Username">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>帳號</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LR_username">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LR_password">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>密碼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LR_Trigger">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>登入/註冊</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/image/TITLE_PAGE.png</pixmap>
      </property>
     </widget>
     <widget class="QRadioButton" name="Login">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>登入</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Register">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>360</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>註冊</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>420</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/image/LANG.png</normaloff>:/image/LANG.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>42</width>
        <height>42</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="LR_Trigger_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>略過登入</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>500</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>程式狀態</string>
      </property>
     </widget>
     <widget class="QLabel" name="LR_status">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>500</y>
        <width>271</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>361</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Version : DEV 0.1</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
